central/policy/datastore/datastore_impl.go

```diff
@@ -451,10 +451,10 @@ func (ds *datastoreImpl) validateUniqueNameAndID(ctx context.Context, policy *st
 		// Similarly, new policy cannot have the same name as default policies
 		if existingPolicy.GetIsDefault() {
 			importErrors = append(importErrors,
-				duplicateNameImportErrf(policiesPkg.ErrImportDuplicateSystemPolicyName, policy.GetName(), "system policy with name %q already exists, unable to import policy", policy.GetId()))
+				duplicateNameImportErrf(policiesPkg.ErrImportDuplicateSystemPolicyName, policy.GetName(), "system policy with id %q already exists, unable to import policy", policy.GetId()))
```

Comment:
I think this is OK because the name will be provided in the metadata of the error.

```
&v1.ImportPolicyError{
		Message: fmt.Sprintf(errMsgTemplate, args...),
		Type:    errType,
		Metadata: &v1.ImportPolicyError_DuplicateName{
			DuplicateName: duplicateName,
		},
```